qbraid/devices/tests/test_results.py

```diff
@@ -41,3 +43,37 @@ def test_format_counts(counts_raw, expected_out):
     counts_out = _format_counts(counts_raw)
     assert counts_out == expected_out  # check equivalance
     assert list(counts_out.items()) == list(expected_out.items())  # check ordering of keys
+
+
+@pytest.mark.skip(
```

Comment:
Instead of `pytest.skip` for this and the test below, can you use the `pytest.skipif` convention? For example: https://github.com/qBraid/qBraid/blob/33728764c03c3ffd18b7eebb8f4e94ffcc32d2ff/qbraid/devices/ibm/tests/test_ibm_provider.py#L41

This way we can conditionally skip if `QBRAID_RUN_REMOTE_TESTS` is not set, and run the test if it is. Environment variable is set to true in our main CI/CD test after a PR is merged.